bikeshed.js

```diff
@@ -41,10 +39,10 @@ class DataBase {
     }
 
     get getItems() {
-        if (this.items) {
-            return this.items;
+        if (!this.items) {
+            return this.items = [];
```

Comment:
Det her fungerer ikke. Du kan nøjes med at bare skrive `get getItems() { return this.items }`

Reply:
Det er fikset nu